lib/event_store/event_store.ex

```diff
@@ -1,7 +1,39 @@
 defmodule Shared.EventStore do
-  defmacro __using__(_opts \\ []) do
-    quote location: :keep, generated: true do
+  defmodule Util do
+    def default_repository(opts, otp_app) do
```

Comment:
How is this repository default?
I understand it to be external or supplied by the user.